sig/mysql_rewinder.rbs

```diff
@@ -5,13 +5,14 @@ class MysqlRewinder
   # @inserted_table_record_dir: Pathname
   @cleaners: Array[Cleaner]
   @inserted_tables: Set[String]
+  @logger: untyped
```

Comment:
Maybe we can fix `untyped` to `Logger` ?

Reply:
The type definition of the logger faced the same issue as described in https://github.com/ruby/rbs/issues/1482, so it was set to untyped.

Reply:
I see. This is OK for now.